### ISSUE TITLE
Add Role based access control

### ISSUE THREAD

**Issue by irwangunardi** [DozerDB/dozerdb-core#31]
Hi, I have been using neo4j community edition for quite a while. However, it is frustrating because the access control only available in Enterprise Edition, and I need the RBAC to create roles with read only privilege, and ability to assign it to users for security and audit purposes.
Is there a plan to implement this in dozer db?

Thanks!

**Comment by jmsuhy** [comment]
Yes, in fact security is next on our list, we are finishing up query logging first.

**Comment by jmsuhy** [comment]
This did not make it into the first 5.26 LTS release - but it was started and will be released soon.   We plan on setting up a community voting system to allow the community to vote on features they want moving forward.

**Comment by irwangunardi** [comment]
Appreciate it @jmsuhy, i'm highly anticipating the next release

**Comment by jjdub7** [comment]
This would actually be awesome for my org's use of community edition.  Thanks for putting it on your radar!

**Comment by rickross** [comment]
+1 on releasing this as soon as you can!
